src/ClimaTimeSteppers.jl

```diff
@@ -66,18 +66,7 @@ include("operators.jl")
 
 abstract type DistributedODEAlgorithm <: DiffEqBase.AbstractODEAlgorithm end
 
-abstract type AbstractIMEXARKAlgorithm <: DistributedODEAlgorithm end
-
-abstract type AbstractTableau end
-abstract type AbstractIMEXARKTableau <: AbstractTableau end
-abstract type AbstractIMEXSSPARKTableau <: AbstractTableau end
-
-"""
-    tableau(::DistributedODEAlgorithm)
-
-Returns the tableau for a particular algorithm.
-"""
-function tableau end
+abstract type AbstractAlgorithmName end
```

Comment:
At the moment, an algorithm is mainly determined by the tableau and constraint (Newton's method is kind of orthogonal). Since the "name" determines both (through the `IMEXTableau` constructor and `default_constraint` function), it feels to me like it should be an "algorithm name".